### PR TITLE
LPS-106853 portal-search-web: Add ADTs for type facet widgets

### PR DESCRIPTION
**Pull request by oliv-yu**
https://issues.liferay.com/browse/LPS-106853

@jonmak08

### CODE REVIEW

**Comment by oliv-yu** [comment]
ci:test:search:norebase

**Comment by oliv-yu** [comment]
ci:test:sf:norebase

File: modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/type/facet/view.jsp [hunk 1]
```diff
@@ -19,85 +19,118 @@
...
-			persistState="<%= true %>"
-		>
-			<liferay-ui:panel
-				collapsible="<%= true %>"
-				cssClass="search-facet"
-				id='<%= renderResponse.getNamespace() + "facetAssetEntriesPanel" %>'
-				markupView="lexicon"
-				persistState="<%= true %>"
-				title="type"
+		<aui:form method="post" name="assetEntriesFacetForm">
```
Comment (jonmak08) [inline modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/type/facet/view.jsp]: Hey @oliv-yu,

For the name of the form, we typically name them `name="fm"` or `name="dialogFm"`.  Can you update and follow a similar pattern?

Please let me know if you have any questions. Thanks!

**Comment by oliv-yu** [comment]
Rebased atop changes in https://github.com/brianchandotcom/liferay-portal/pull/84725 👍

File: modules/apps/portal-search/portal-search-web/src/main/resources/resource-actions/default.xml [hunk 2]
```diff
@@ -142,4 +142,24 @@
 			</guest-unsupported>
 		</permissions>
 	</portlet-resource>
+	<portlet-resource>
+		<portlet-name>com_liferay_portal_search_web_type_facet_portlet_TypeFacetPortlet</portlet-name>
```
Comment (thektan) [inline modules/apps/portal-search/portal-search-web/src/main/resources/resource-actions/default.xml]: Alphabetize

**Comment by oliv-yu** [comment]
ci:test:search:norebase

**Comment by oliv-yu** [comment]
ci:test:sf:norebase

File: modules/apps/portal-search/portal-search-web/src/main/resources/resource-actions/default.xml (hunk 2 above)
Comment (oliv-yu) [reply modules/apps/portal-search/portal-search-web/src/main/resources/resource-actions/default.xml]: Ok! Pushed up a commit to alphabetize, and rebased atop a stable branch w/ cherry-picks.

**Comment by thektan** [comment]
Forwarded to https://github.com/thektan/liferay-portal/pull/148

**Pull request closed without merging**